public/javascripts/SVLabel/spec/SVLabel/modal/ModalMissionCompleteProgressBar-spec.js

```diff
@@ -0,0 +1,107 @@
+describe("ModalMissionCompleteProgressBar tests", function () {
+	var bar;
+
+	beforeEach( function () {
+		$uiModalMissionCompleteFixture = $('<div id="modal-mission-complete-complete-bar"></div>');
+		$('body').append($uiModalMissionCompleteFixture);
```

Comment:
I am not sure how else to do it with the current way the modules are set up. They search for a tag in the html and I dont know how else it would work